app/src/main/java/app/suhasdissa/mellowmusic/backend/services/PlayerService.kt

```diff
@@ -62,7 +70,19 @@ class PlayerService : MediaSessionService(), MediaSession.Callback, Player.Liste
         }
         cache = SimpleCache(directory, cacheEvictor, StandaloneDatabaseProvider(this))
 
-        player = ExoPlayer.Builder(this, createRendersFactory(), createMediaSourceFactory())
+        player = createPlayer()
+
+        player.repeatMode = Player.REPEAT_MODE_OFF
+        player.playWhenReady = true
+        player.addListener(this)
+
+        mediaSession = MediaSession.Builder(this, player).setCallback(this)
+            .setBitmapLoader(CustomBitmapLoader(this)).build()
```

Comment:
```suggestion
            // .setBitmapLoader(CustomBitmapLoader(this))
            .build()
```

This code is causing the app to crash when playing local music.
Let's remove this for now.

Reply:
Oh okay, it worked for me because I didn't test any music with a thumbnail.